packages/ui/cypress/tests/network-switch.cy.ts

```diff
@@ -0,0 +1,44 @@
+import { accountDisplay } from '../support/page-objects/components/accountDisplay'
+import { settingsPageWatchAccountUrl } from '../fixtures/landingData'
+import { settingsPage } from '../support/page-objects/settingsPage'
+import { testAccounts } from '../fixtures/testAccounts'
+import { topMenuItems } from '../support/page-objects/topMenuItems'
+
+const {
+  name: testAccountName,
+  address: testAccountAddress,
+  kusamaAddress
+} = testAccounts['Multisig Member Account 1']
+
+describe('Network can be switched', () => {
+  it('should switch account using menu', () => {
+    cy.visitWithLocalStorage({
+      url: settingsPageWatchAccountUrl,
+      accountNames: { [testAccountAddress]: testAccountName as string },
+      watchedAccounts: [testAccountAddress]
```

Comment:
Please use the public key here instead of the address, as done in all other tests. The fact that it works with an address is actually a legacy property that could be removed. The UI stores the watched accounts in a way that it's not tied to the network.